docs/blog/2020-01-30-why-gatsby-is-better-with-javascript/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Why Gatsby is better with JavaScript"
+date: 2020-01-30
+author: Mikhail Novikov
+excerpt: "Gatsby sites can work without JavaScript, however they will often be faster when JavaScript is enabled. In addition, adding interactive elements is much easier."
+tags:
+  - Gatsby
+  - JavaScript
+  - Performance
+  - Accessibility
+---
+
+Gatsby sites can work without JavaScript. Because Gatsby renders all the pages as plain HTML files, users can use the majority of a website's functionality without it. JavaScript is required only for additional interactive elements of an app.
+
+We have often been asked to add an option to not build or include JavaScript bundles in a site. However we believe that this will make Gatsby performance worse, not better, and this blog post aims to explain why we think this is the case. We will also cover how some future React features, like partial hydration, will improve Gatsby performance.
```

Comment:
```suggestion
We're often asked to add an option to not build or include JavaScript bundles in a site. However we believe that this will make Gatsby performance worse, not better, and this blog post aims to explain why we think this is the case. We will also cover how some future React features, like partial hydration, will improve Gatsby performance.
```